native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -25,6 +28,14 @@ class ViewController: UIViewController {
         arrivalStationLabel.text = "Arrival"
         label.text = "TrainBoard"
     }
+    
+    func findStationWithName(name: String) -> Station? {
+        guard let index = stationData.firstIndex(where: { $0.name.lowercased() == name.lowercased() }) else {
```

Comment:
I think we should also trim any whitespace from the start and end of the string, i.e.

`$0.name.trimmingCharacters(in: .whitespacesAndNewlines).lowercased()`

I'm going to open a new PR to fix this on android, but can be captured as part of this PR on iOS.

Reply:
done